### ISSUE TITLE
Error while running command (com.milaboratory.mixcr.cli.CommandAnalyze$CommandAmplicon@7c18432b): java.lang.ArrayIndexOutOfBoundsException

### ISSUE THREAD

**Issue by Kaj-cyber** [milaboratory/mixcr#664]
## Checklist before submitting the issue:

- [x] The issue is strongly related to the MiXCR software

 - [x] The issue can be reproduced with [the most recent version](https://github.com/milaboratory/mixcr/releases) of MiXCR
 - [x] There is no answer to the question in the [official documentation](http://mixcr.read
[imgt.zip](https://github.com/milaboratory/mixcr/files/7141980/imgt.zip)
thedocs.io/) and there is no duplicate issue in the [bug tracker](https://github.com/milaboratory/mixcr/issues?utf8=✓&q=)  
 - [x] Inspection of raw alignments with [exportAlignmentsPretty](http://mixcr.readthedocs.io/en/master/export.html#exporting-well-formatted-alignments-for-manual-inspection) shows that data has the expected architecture, and sample preparation artefacts are not the reason of the problem (if this is the matter of the issue)

## Problem

Dear all great helpers,

I compiled json library for cat's TCR using 'repseqio' as attached. After testing it with the cat's TCR miSeq data, I got the error as indicated in this question. Would you please help me fix the issue?

Best regards,
Kaj

## Expected Result

Successful alignment and assemble

## Actual Result

The error as following:
```

picocli.CommandLine$ExecutionException: Error while running command (com.milaboratory.mixcr.cli.CommandAnalyze$CommandAmplicon@7c18432b): java.lang.ArrayIndexOutOfBoundsException: Index 19 out of bounds for length 19
        at picocli.CommandLine.execute(CommandLine.java:1004)
        at picocli.CommandLine.access$900(CommandLine.java:142)
        at picocli.CommandLine$RunLast.handle(CommandLine.java:1199)
        at com.milaboratory.mixcr.cli.Main$1.handle(Main.java:76)
        at com.milaboratory.mixcr.cli.Main$1.handle(Main.java:65)
        at picocli.CommandLine$AbstractParseResultHandler.handleParseResult(CommandLine.java:1075)
        at com.milaboratory.mixcr.cli.Main.handleParseResult(Main.java:89)
        at com.milaboratory.mixcr.cli.Main.main(Main.java:59)
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 19 out of bounds for length 19
        at io.repseq.core.AbstractReferencePoints.getWrappingGeneFeature(AbstractReferencePoints.java:198)
        at io.repseq.core.VDJCLibrary.getChecksum(VDJCLibrary.java:89)
        at io.repseq.core.VDJCLibrary.getLibraryId(VDJCLibrary.java:206)
        at com.milaboratory.mixcr.cli.CommandAlign.getConfiguration(CommandAlign.java:308)
        at com.milaboratory.mixcr.cli.CommandAlign.getFullPipelineConfiguration(CommandAlign.java:299)
        at com.milaboratory.cli.ACommandWithSmartOverwrite.handleExistenceOfOutputFile(ACommandWithSmartOverwrite.java:76)
        at com.milaboratory.cli.ACommandWithOutput.validate(ACommandWithOutput.java:45)
        at com.milaboratory.cli.ACommandWithSmartOverwrite.validate(ACommandWithSmartOverwrite.java:61)
        at com.milaboratory.mixcr.cli.CommandAlign.validate(CommandAlign.java:382)
        at com.milaboratory.mixcr.cli.CommandAnalyze.inheritOptionsAndValidate(CommandAnalyze.java:320)
        at com.milaboratory.mixcr.cli.CommandAnalyze.getAlign(CommandAnalyze.java:338)
        at com.milaboratory.mixcr.cli.CommandAnalyze.run0(CommandAnalyze.java:668)
        at com.milaboratory.cli.ACommand.run(ACommand.java:90)
        at picocli.CommandLine.execute(CommandLine.java:996)
        ... 7 more


```
## Exact MiXCR commands 
```

$MIXCR analyze amplicon \
--threads 20 \
--library imgt \
--species "cat" \
--starting-material rna \
--5-end v-primers \
--3-end j-primers \
--adapters adapters-present \
--receptor-type tcr \
--region-of-interest VDJRegion \
--only-productive \
--align "-OreadsLayout=Collinear" \
--assemble "-OseparateByC=true" \
--assemble "-OqualityAggregationType=Average" \
--assemble "-OclusteringFilter.specificMutationProbability=1E-5" \
--assemble "-OmaxBadPointsPercent=0" \
pSRR10350408_1.fq pSRR10350408_2.fq test.analysis


```
## MiXCR report files 

The empty analysis file was produced.

**Comment by Kaj-cyber** [comment]
I found this might be due to my TCRD library compilation, and hereby asked this question in 'repseqio'. I must apologize for any inconvenience I might cause to you.

**Comment by zxl215** [comment]
@Kaj-cyber Hi, I am  having the same issue. Have you solved the issue? Could you help me?

**Comment by Kaj-cyber** [comment]
> @Kaj-cyber Hi, I am having the same issue. Have you solved the issue? Could you help me?
@zxl215 
As far as I could remember, I think this is possibly due to redundancy and/or incomplete of the TCR sequences (the region was lack of or had several reported sequences). I remembered that I decided to use only the first sequence reported for the redundant ones. I also separately complied only TRA, TRB, TRD and TRG and merge them into a single library (I could share the codes, but I couldn't exactly remember what I had done).

Sorry for my not so helpful answer
